server/game/cards/events/01/thehandsjudgment.js

```diff
@@ -28,7 +28,7 @@ class TheHandsJudgment extends DrawCard {
             return super.getCost();
         }
 
-        return this.eventToInterrupt.getCost();
+        return this.eventToInterrupt.cardData.cost || 0;
```

Comment:
This is literally what getCost() does, except the || 0 bit, so do we not need to fix it in DrawCard instead of here?